src/www/services_unbound_advanced.php

```diff
@@ -133,64 +133,64 @@
                       <tr>
                         <td><a id="help_for_hideversion" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Hide Version");?></td>
                         <td>
-                          <input name="hideversion" type="checkbox" id="hideversion" value="yes" <?=!empty($pconfig['hideversion']) ? "checked=\"checked\"" : "";?> />
```

Comment:
@fabianfrz I don't mind switching the order, but don't see the big gain (issues slip in very easily). For the future, can you keep the order as it is if the result is good? Thanks in advance.